### ISSUE TITLE
Generation of corrected z-score values

### ISSUE THREAD

**Issue by shirondru** [Illumina/PromoterAI#11]
Hi there, 

I was wondering if you'd be willing to share some code for the generation of corrected z scores as well as the t-test performed to call outliers. Anything at all would be very helpful. Thanks in advance for anything you can share!

**Comment by shirondru** [comment]
Just following up on this. If you're unable to share code, would it be possible to elaborate on the z-scoring procedure described in the methods of the paper? 

My expectation is that the goal was to generate Z scores for each individual, per gene and per tissue, such that the value would represent the individual's deviation from the average individual's expression for a given gene in a given tissue. But in the "Generation of corrected expression z-scores across tissues" section, all re-scaling events are described as "values were re-scaled within tissue by both gene and individual". I interpret this to mean that the z-score calculation for each tissue is computed using the mean and standard deviation value computed across all genes and all individuals, which would give a different interpretation to any expression value than the one I described. Could you please clarify?

Thanks again for all your help!

**Comment by MiqG** [comment]
Hi, regarding this, I also have a question on the correction for trans effects. In your methods section you explain: 

*"The model consisted of one fully connected hidden layer of size 1,024 and was trained using an MSE loss with batch size of 32, patience of 20, dropout rate of 0.1, and normal initialization of model weights. The model was trained for a maximum of 100 epochs and was implemented in TensorFlow v2.3.0."*

Was the neural net purely linear or was any activation function applied (e.g., ReLU)?

Thanks! Best,

Miquel

**Comment by kishorejaganathan** [comment]
ReLU activation function
